package.json

```diff
@@ -9,6 +9,9 @@
     "Lufty Wiranda <lufty.wiranda@gmail.com>",
     "Suhas Karanth <sudo.suhas@gmail.com>"
   ],
+  "engines": {
+    "node": "^8.12.0 || >=9.7.0"
```

Comment:
We didn’t include that on purpose after a discussion and instead we check for the minimum node version in runtime

Reply:
Why do we need the `||` here and not just `>=` anyways?

Reply:
I copy-pasted this from `execa` since it was the most strict requirement of the dependencies. I have no opinion about this, maybe it’s best to not include it in this case.

Reply:
Let’s not do this then. We need to check what version is required in “please-upgrade-node”, though.